contracts/auction/ClockAuctionBase.sol

```diff
@@ -145,7 +152,8 @@ contract ClockAuctionBase {
             // before calling transfer(), and the only thing the seller
             // can DoS is the sale of their own asset! (And if it's an
             // accident, they can call cancelAuction(). )
-            seller.transfer(sellerProceeds);
+            // TODO: change this to RING
+            RING.transfer(seller, sellerProceeds);
```

Comment:
Should use ERC223 features to get RING transferred to this contract before this callback.